README.md

```diff
@@ -55,6 +55,60 @@ tools such as **swift-format**, **swiftlint**, **xcodebuild**, **xcrun**, **xcod
 1. 🍏 MacOS — other platforms are currently not supported
 2. 📱 Xcode — required for building and running iOS apps via `xcodebuild`
 
+## Development
+
+### 🛠️ **Local Development Setup**
+
+If you want to contribute to SweetPad or test changes locally, you can use the provided installation script:
+
+```bash
+# Clone the repository
+git clone https://github.com/sweetpad-dev/sweetpad.git
+cd sweetpad
+
+# Install dependencies
+npm install
+
+# Build and install the extension locally
+./scripts/install-and-test.sh
```

Comment:
I usually build and install the extension by opening "sweetpad" repo in VS Code and by pressing F5 . It opens a separate VS Code window with auto-reloading and debugging enabled. Could you try that option? I think it might be an easier way to start contributing

https://github.com/user-attachments/assets/5be23588-bc95-4b54-a715-efe263df0b13